src/components/CalendarMonthGrid.jsx

```diff
@@ -54,23 +57,26 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY', // english locale
 };
 
-function getMonths(initialMonth, numberOfMonths) {
-  let month = initialMonth.clone().subtract(1, 'month');
-
-  const months = [];
-  for (let i = 0; i < numberOfMonths + 2; i++) {
-    months.push(month);
-    month = month.clone().add(1, 'month');
+export function getMonths({ initialMonth, numberOfMonths, addTransitionMonths }) {
```

Comment:
prefer if you're exporting a function for tests that you separate this out into its own file.